iceoryx_examples/ice_multi_publisher/ice_resubscriber.cpp

```diff
@@ -35,10 +35,11 @@ constexpr uint64_t UNSUBSCRIBED_TIME_SECONDS{3U};
 
 void receive()
 {
-    iox::popo::TypedSubscriber<CounterTopic> subscriber({"Group", "Instance", "Counter"});
+    iox::popo::SubscriberOptions options;
+    options.queueCapacity = MAX_NUMBER_OF_SAMPLES - 2U;
```

Comment:
I feel this makes no more sense as the queue size is no more adapted. So I would remove `MAX_NUMBER_OF_SAMPLES` completely or use it to set the capacity but without the magic -2

Reply:
maybe the whole example is obsolete? Do we really need to show how to repeatedly call `subscribe` and `unsubscribe`?

Reply:
That's true. You could even drop this and adapt the README.

Reply:
can I do this in a follow up PR? I also need to adjust the example to show how to use the history feature and this PR is already quite large